src/ManagedServiceIdentity/ManagedServiceIdentity.Test/ScenarioTests/CrudTests.ps1

```diff
@@ -62,6 +62,7 @@ function Test-CrudUserAssignedIdentity
 
         #Delete Identity1
         Remove-AzUserAssignedIdentity -ResourceGroupName $rgName1 -Name $identityName1 -Force;
+        Start-Sleep -s 5
```

Comment:
There is a common cmdlet called Wit-Seconds that you shoudl use - however this should never be done without a test to ensure the condition you are watiting for has occurred.